cristin-person/src/test/java/no/unit/nva/cristin/person/employment/delete/personDeleteEmployment.feature

```diff
@@ -0,0 +1,65 @@
+Feature: API tests for Cristin Person fetch
+
+  Background:
+    * def domainName = java.lang.System.getenv('DOMAIN_NAME')
+    * def basePath = java.lang.System.getenv('BASE_PATH')
+    * def CRISTIN_BASE =  'https://' + domainName +'/' + basePath
+    * def username = java.lang.System.getenv('TESTUSER_FEIDE_ID')
+    * def password = java.lang.System.getenv('TESTUSER_PASSWORD')
+    * def cognitoClientAppId = java.lang.System.getenv('COGNITO_CLIENT_APP_ID')
+    * def cognitoUserpoolId = java.lang.System.getenv('COGNITO_USER_POOL_ID')
+    * def tokenGenerator = Java.type('no.unit.nva.cognito.CognitoUtil')
+    * def token = tokenGenerator.loginUser(username, password, cognitoUserpoolId, cognitoClientAppId)
+    * def nonExistingPersonId = '100000000'
+    * def nonExistingEmploymentId = '100000000'
+    * def nonValidPersonId = '000A000'
+    * def nonValidEmploymentId = '000B0000'
+    * def existingPersonId = '538786'
+    * def existingEmploymentId = '49272'
+    * def invalidToken = 'just-a-invalid-token-for-now'
+    Given url CRISTIN_BASE
+    * print 'Current base url: ' + CRISTIN_BASE
+
+  Scenario: Delete returns status 401 Forbidden when called without valid token
+    Given path '/person/' + nonExistingPersonId +'/employment/' + nonExistingEmploymentId
+    * header Authorization = 'Bearer ' + invalidToken
+    When method DELETE
+    Then status 401
```

Comment:
Karate test for missing token and valid token without required rights or is that redundant?

Reply:
It's redundant I guess, all we care about is the presence of access right - not the token itself